Sources/CardPayments/Models/Card.swift

```diff
@@ -22,7 +22,7 @@ public struct Card {
     public var billingAddress: Address?
     
     /// The expiration year and month, in ISO-8601 `YYYY-MM` date format.
-    private var expiry: String {
+    public var expiry: String {
```

Comment:
qq: What happens if we keep this private?

Reply:
I changed it so I could copy over expiry from Card to VaultCard.